### ISSUE TITLE
Slow to play MP3s when no internet connection?

### ISSUE THREAD

**Issue by PaperOrb** [tagspaces/tagspaces#1676]
**Describe the bug**
Clicking an mp3 file takes 30+ seconds before tagspaces free version starts up its inbuilt audio player if there's no internet connection.

**Expected behavior**
Internet connection shouldn't affect mp3 playback speed inside the program.

**Desktop Application:**
 - Ubuntu 21.04
 - TagSpaces Version 4.4.3

**Comment by uggrock** [comment]
I can not reproduce this issue on my side. Playing mp3 is entirely offline process, with no need of Internet at all. Can you please restart your computer and try to reproduce the issue. Maybe some other background process is slowing down your PC.

**Comment by sytolk** [comment]
My OS: Windows 10 and I cannot repeat this issue. Song is playing fast with disabled WiFi connection. 
Maybe the problem is with the song file? @PaperOrb can you provide the song file ?

**Comment by PaperOrb** [comment]
> My OS: Windows 10 and I cannot repeat this issue. Song is playing fast with disabled WiFi connection. Maybe the problem is with the song file? @PaperOrb can you provide the song file ?
It's important to follow the steps I listed in order to reproduce this. I noticed if I had tagspaces open and then turned wifi off, there was no delay when playing songs. 

Also this wasn't just one song file, but a dozen different ones with several formats--it's very unlikely it's an issue with the files themselves. I haven't tested this on windows, so I can't say if it's the appimage or not that's the issue.
